Bottom-Up ET

balakumaran247 opened the pull request:
# pAnnualDaytimeHrs.py
- created FAO p value for different latitudes referenced from a source file.
- this script takes two args - var_type (tmax or tmin) and filename (tif file) for location of reference file

# IMDHistoricalGrid.py
- I added a line to check whether the output directory exists or will create one if not

craigdsouza commented:
Hey can you put the p value script in this path "scripts/evapotranspiration/fao"?

balakumaran247 commented:
# commit fbd3ca0
- Moved pAnnualDaytimeHrs script to path 'scripts/evapotranspiration/fao'

# commit 4d88fff
- Added SSEBopMonthlyGCP2GEE.py script

# commit a9c54b7
- closed the imagery after reading it using rasterio.close() 
- pAnnualDaytimeHrs.py, getWaterYrSEBOP.py, IMDHistoricalDailySum.py, IMDHistroicalMeanTemp.py, effPrecipitation.py scripts were corrected to include closing of the opened dataset.

# commit 40a190a
- Updated output path to {Home Dir}\Data\et\fao\p_value\
